content/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file.md

```diff
@@ -37,6 +37,7 @@ Community health file | Description
 *CONTRIBUTING.md* | A CONTRIBUTING file communicates how people should contribute to your project. For more information, see "[AUTOTITLE](/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)."{% ifversion discussion-category-forms %}
 Discussion category forms | Discussion category forms customize the templates that are available for community members to use when they open new discussions in your repository. For more information, see "[AUTOTITLE](/discussions/managing-discussions-for-your-community/creating-discussion-category-forms)."{% endif %}{% ifversion fpt or ghec %}
 *FUNDING.yml* | A FUNDING file displays a sponsor button in your repository to increase the visibility of funding options for your open source project. For more information, see "[AUTOTITLE](/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository)."{% endif %}
+*GOVERNANCE.md* | A GOVERNANCE file lets people know about how your project is governed. For examples, a sole maintainer project; a small group project that operates by vote; or a group-run project that is interested in adding new governance leaders over time."
```

Comment:
```suggestion
*GOVERNANCE.md* | A GOVERNANCE file lets people know about how your project is governed. For example, it might discuss project roles and how decisions are made.
```